src/test/datascience/notebook/helper.ts

```diff
@@ -529,10 +529,31 @@ export async function waitForExecutionInProgress(cell: NotebookCell, timeout: nu
 export async function waitForQueuedForExecution(cell: NotebookCell, timeout: number = defaultTimeout) {
     await waitForCondition(
         async () => {
-            return NotebookCellStateTracker.getCellState(cell) === NotebookCellExecutionState.Pending;
+            return (
```

Comment:
Isn't this function still just Pending? It looks the same as the condition below.